examples/torch_tensorrt/resnet_tensorrt.py

```diff
@@ -0,0 +1,19 @@
+import torch
```

Comment:
could you put this in our tests instead?

However as a UX I believe TensorRT will soon instead have a dynamo backend which means we wont need a separate integration for this. See this https://github.com/pytorch/pytorch/pull/104426

Reply:
Thats great news. Since the release will be in october , and we'll have a TS release after that,  we can have this till then and then update it?

For  unit tests, the challenge is getting torch tensorrt to work. We need cudnn, I could get it working with a specific combination. Even Torch TensorRT's prescribed way is using nvidia's docker container.